packages/ui/react-ui-table/src/components/Table/Table.stories.tsx

```diff
@@ -323,15 +323,15 @@ export const InsertDelete = {
     };
 
     return (
-      <div>
+      <div className='flex flex-col gap-4'>
```

Comment:
```suggestion
      <div className='space-y-4'>
```
Should avoid `flex` for these use-cases, it has caused subtle bugs in the past.